src/main/java/equipments/EquipmentType.java

```diff
@@ -0,0 +1,9 @@
+package equipments;
+
+public enum EquipmentType {
+    VOIP,
+    InternetComputer,
```

Comment:
if you use `EquipmentType.toString()` (implicitly, in the describer), give the values a more user-friendly name so that the final value isn't in camelCase